src/components/profile/profile-logged-in/adopted-trees/adopted-trees.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { useI18nStore } from "../../../../i18n/i18n-store";
+import { TertiaryButton } from "../../../buttons/tertiary";
+import { TreeCard } from "./tree-card";
+import { useAuthStore } from "../../../../auth/auth-store";
+import { AdoptedTreeIcon } from "../../../icons/adopted-tree-icon";
+import { Skeleton } from "../../../skeleton/skeleton";
+import { AdoptedTreesCard } from "./adoptedTreesCard";
+
+export const AdoptedTrees: React.FC = () => {
+	const i18n = useI18nStore().i18n();
+	const [showAllTrees, setshowAllTrees] = useState(false);
```

Comment:
one last comment, let's use here `areAllTreesVisible` to keep it consistent with other booleans (is/are/has pattern)